bug_report improvements (#7805)
netdata/netdata: .github/ISSUE_TEMPLATE/bug_report.md

labels: bug, needs triage
---

<!--
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub.
- Test if the latest release and master branch are affected too.
-->

##### Bug report summary
<!-- Provide a clear and concise description of what the bug. -->

##### OS / Environment
<!--
Provide as much information about your environment (OS distribution, running in container, etc.)
as possible to allow us reproduce this bug faster.

To get this information execute:
- uname -a; grep -Hv "^#" /etc/*release  # linux/bsd
- uname -a; sw_vers                      # macOS

> 

Reviewer: The previous 3 lines should be on `OS /Environment`

Reviewer: Sorry, I only saw now that you are removing Netdata version.

Author: i havent remove anything, i moved stuff to appropriate section.

> The previous 3 lines should be on OS /Environment

Those limes under `OS /Environment` section.

labels: bug, needs triage
---

<!--
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub.
- Test if the latest release and master branch are affected too.
-->

##### Bug report summary
<!-- Provide a clear and concise description of what the bug. -->

##### OS / Environment
<!--
Provide as much information about your environment (OS distribution, running in container, etc.)
as possible to allow us reproduce this bug faster.

To get this information execute:
- uname -a; grep -Hv "^#" /etc/*release  # linux
- uname -a; uname -K                     # bsd

> - uname -a; sw_vers                      # macOS

Reviewer 2: I _think_ that should work on all modern BSD variants, but we technically only support FreeBSD right now.

> Place output in the code section.  

Reviewer 3: ```suggestion
Place the output from the command in the code section below.
```

 -->
```

```

##### Netdata version
<!--
Provide output of netdata -V.
 

> If netdata is running execute: $(ps aux | grep -E -o "[a-zA-Z/]+netdata ") -V

Reviewer 3: ```suggestion
If Netdata is running, execute: $(ps aux | grep -E -o "[a-zA-Z/]+netdata ") -V
```

 -->
 

##### Component Name
<!--
Write which component is affected. We group our components the same way our code is structured so basically: 
component name = dir in top level directory of repository.

> -->

Reviewer 3: Let's try something like this:

Let us know which component is affected by the bug. Our code is structured according to its component, so the component name is the same as the top level directory of the repository. For example, a bug in the dashboard would be under the web component.

##### Steps To Reproduce
<!--

> Describe how you found this bug and how we can reproduce it. Preferable with a minimal test-case scenario.

Reviewer 3: ```suggestion
Describe how you found this bug and how we can reproduce it, preferably with a minimal test-case scenario.
```

> You can paste gist.github.com links for larger files.

Reviewer 3: ```suggestion
If you'd like to attach larger files, use gist.github.com and paste in links.
```
